src/proj_internal.h

```diff
@@ -677,8 +677,8 @@ struct FACTORS {
 #define PJD_ERR_TOO_MANY_INITS          -57
 #define PJD_ERR_INVALID_ARG             -58
 #define PJD_ERR_INCONSISTENT_UNIT       -59
-/* NOTE: Remember to update pj_strerrno.c and transient_error in */
-/* pj_transform.c when adding new value */
+/* NOTE: Remember to update strerrno.cpp and transient_error in */
+/* transform.cpp when adding new value */
```

Comment:
gie.cpp also need to be added to this list (incidentally I did the same correction as you in https://github.com/OSGeo/proj.4/pull/1250/)

Reply:
Done